src/app/phast/losses/wall-losses/wall-losses-form/wall-losses-form.component.ts

```diff
@@ -54,7 +52,7 @@ export class WallLossesFormComponent implements OnInit {
     }
     this.surfaceOptions = this.suiteDbService.selectWallLossesSurface();
     //init warnings
-    this.checkWarnings();
+    // this.checkWarnings();
```

Comment:
Commented out. Looks like function is deleted. Output "inputError" no longer used